lib/aur-sync

```diff
@@ -82,7 +82,7 @@ complement() {
 list_markers_then_files() {
     find -L "$1" -maxdepth 1 -mindepth 1 -name '!*.new'
     find -L "$1" -maxdepth 1 -mindepth 1 -name '!*.patch' -o -name '!*.diff'
-    find -L "$1" -maxdepth 2 -mindepth 1 ! \( -ipath "$1/!*.new" -o -ipath "$1/!*.patch" -o -ipath "$1/!*.diff" \)
+    find -L "$1" -maxdepth 2 -mindepth 1 -not \( -ipath "$1/\!*.new" -o -ipath "$1/\!*.patch" -o -ipath "$1/\!*.diff" \)
```

Comment:
another small note, `-not` and `-ipath` are not POSIX.
They could be replaced by `!` and `-path`. 
@AladW what's your position on this?